tsconfig.json

```diff
@@ -10,19 +10,21 @@
         "outDir": "dist",
         "sourceMap": true,
         "resolveJsonModule": true,
-        "esModuleInterop": true
+        "esModuleInterop": true,
     },
     "include": [
         "src",
         ".env.dev",
         ".env.prod",
         "jest.config.js",
         "__tests__/server.test.ts",
-        "utils/environmentVariables.ts",
-        "utils/typesgeneratorconfig.ts",
-        "utils/databaseSeedTool.ts"
+        "src/typeDefs/schema.graphql",
+        "src/utils/environmentVariables.ts",
+        "src/utils/typesgeneratorconfig.ts",
+        "src/utils/databaseSeedTool.ts"
```

Comment:
Could we simplify this to `src/utils/*`?

Reply:
this isn't ready for review yet. I'm messing around with this file at the moment because it won't seem to exclude/include files in this list. 🤔